src/app/pages/wallets/my-wallets/pages-wallets-my-wallets.component.html

```diff
@@ -4,7 +4,7 @@
   </div>
 
   <div *ngIf="myWalletsData.hasError">
-    ERROR - We need to implement some fancy error component
+    <app-system-message type="error" text="Something went wrong ;(" description="Some server error occured"></app-system-message>
```

Comment:
Lets tune it up a bit with:
- text "Something went wrong. We were not able to get your wallets"
- description "Please try refreshing your browser"